ext/hash/hash_murmur.c

```diff
@@ -36,9 +36,20 @@ const php_hash_ops php_hash_murmur3a_ops = {
 	0
 };
 
-PHP_HASH_API void PHP_MURMUR3AInit(PHP_MURMUR3A_CTX *ctx)
+PHP_HASH_API void PHP_MURMUR3AInit(PHP_MURMUR3A_CTX *ctx, HashTable *args)
 {
-	ctx->h = 0;
+	if (args) {
+		zval *seed;
+		/* This might be a bit too restrictive, but thinking that a seed might be set
+			once and for all, it should be done a clean way. */
+		if ((NULL != (seed = zend_hash_str_find(args, "seed", sizeof("seed") - 1))) && IS_LONG == Z_TYPE_P(seed)) {
```

Comment:
```suggestion
		if ((NULL != (seed = zend_hash_str_find_deref(args, "seed", sizeof("seed") - 1))) && IS_LONG == Z_TYPE_P(seed)) {
```

Reply:
Though, as a style nit, I would write this:
```
zval *seed = zend_hash_str_find_deref(args, "seed", sizeof("seed") - 1);
if (seed && Z_TYPE_P(seed) == IS_LONG)`
```
to avoid having such a large condition.

Reply:
Fixed as suggested, thanks!